### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by jlecomte**
In this PR, I added unit tests, health checks, Travis config and Travis badge.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+    - "0.10"
```
Comment (juandopazo) [inline .travis.yml]: I think 0.12 is already in Travis.

File: tests/unit/lib.layouts.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*global describe, it*/
+
+process.env.NODE_ENV = 'production';
```
Comment (juandopazo) [inline tests/unit/lib.layouts.js]: What's this for?

**Comment by jlecomte** [comment]
Note that these tests would be wholly inadequate for a consumer facing application that generates revenue. However, for the pure site, I think that this may be good enough to give us enough confidence to implement CD.

Also note that I struggle with the testing of the routes. My tests look more like acceptance tests than true unit tests. They are however not redundant with the health checks, which serve a slightly different purpose.

Finally, note that we may need to add unit tests for additional libraries (the stuff under lib/) but I haven't had time to implement those, and I don't want this task to turn into a multi-week effort. I may, however, send subsequent PRs to fill those holes as time allows.

File: tests/unit/lib.layouts.js (hunk 2 above)
Comment (jlecomte) [inline tests/unit/lib.layouts.js]: There are checks on this env variable throughout the code. This just makes things a little easier for me.

**Comment by juandopazo** [comment]
Looks about right. I have some extra unit tests for everything in lib/ I'll send if I manage to dig them up.

**Comment by jlecomte** [comment]
@juandopazo that would be awesome if you could send a separate PR for that stuff, if you can dig it out.

File: tests/unit/lib.layouts.js (hunk 2 above)
Comment (ericf) [inline tests/unit/lib.layouts.js]: This could be moved to shell command that kicks off the tests.

**Pull request merged**